### ISSUE TITLE
nltk.text.Text.dispersion_plot() fails with ImportError: cannot import name 'dispersion_plot'

### ISSUE THREAD

**Issue by awhan** [nltk/nltk#1394]
How to reproduce?

```
from nltk.book import *
text4.dispersion_plot(["citizens", "democracy", "freedom", "duties", "America"])
```

Error log in IPython shell:

```
In [15]: text4.dispersion_plot(["citizens", "democracy", "freedom", "duties", "America"])
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-15-d98be29a81e2> in <module>()
----> 1 text4.dispersion_plot(["citizens", "democracy", "freedom", "duties", "America"])

/usr/lib/python3.5/site-packages/nltk/text.py in dispersion_plot(self, words)
    442         :seealso: nltk.draw.dispersion_plot()
    443         """
--> 444         from nltk.draw import dispersion_plot
    445         dispersion_plot(self, words)
    446

ImportError: cannot import name 'dispersion_plot'
```

Versions:

```
In [18]: nltk.__version__
Out[18]: '3.1'

$ python -V
Python 3.5.1
$ uname -a
Linux 4.5.2-1-ARCH #1 SMP PREEMPT Thu Apr 21 18:21:27 CEST 2016 x86_64 GNU/Linux
```

**Comment by pkubik** [comment]
Same here. You can get around by using the dispersion_plot directly.

``` python
from nltk.draw.dispersion import dispersion_plot
dispersion_plot(text4, ["citizens", "democracy", "freedom", "duties", "America"])
```

**Comment by pkubik** [comment]
Ok. It seems to be fixed on the development branch. The fix came in the #1364 pull request on 2016-04-17 so it might be still unavailable on some distributions. The NLTK version did not change since this fix (3.2.1) so there is no hint when looking for a proper version other than cloning the repo. The best solution would be to simply install the TKinter module.
